packages/foreman/foreman-installer/foreman-installer.spec

```
@@ -56,7 +60,8 @@ Various scenarios and tools for the Katello ecosystem
 %if %{?scl:1}%{!?scl:0}
```

Can be simplified as well

```suggestion
%if 0%{?scl:1}
```